src/main/java/com/bullhorn/dataloader/util/TemplateUtil.java

```diff
@@ -54,14 +55,33 @@ private void writeToCsv(String entity, ArrayList<String> headers, ArrayList<Stri
     }
 
     protected void populateDataTypes(String entity, Set<Field> metaFieldSet, ArrayList<String> headers, ArrayList<String> dataTypes) {
+        try {
+            final ClassLoader classLoader = getClass().getClassLoader();
+            final Class entityClass = classLoader.loadClass("com.bullhornsdk.data.model.entity.core.standard." + entity);
```

Comment:
This use of a try...catch block is unnecessary.  We are already validating the input in the TemplateService, and the TemplateUtil can assume that it is receiving valid data.